integration_tests/can_launch/test_driver/main_test.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:flutter_driver/flutter_driver.dart';
```

Comment:
I don't like it either. Only picked it so it alphabetically appears next to `text`. What do you think of `test_e2e` or `test_integration`? The long-term goal is to switch to `@TestOn` annotation, put all test files under `test/`, and remove this directory.